charts/line-chart/src/LineChart.stories.tsx

```diff
@@ -2,16 +2,29 @@ import React from 'react';
 import { storybookArgTypes } from '@lg-tools/storybook-utils';
 import { StoryFn } from '@storybook/react';
 
-import { LineChart } from '.';
+import LeafyGreenProvider from '@leafygreen-ui/leafygreen-provider';
+
+import { LineChart } from './LineChart';
+import { makeData } from './utils';
 
 export default {
   title: 'Charts/LineChart',
   component: LineChart,
   argTypes: {
     darkMode: storybookArgTypes.darkMode,
   },
+  decorator: (Instance, context) => {
+    return (
+      <LeafyGreenProvider darkMode={context?.args.darkMode}>
+        <Instance darkMode={context?.args.darkMode} />
+      </LeafyGreenProvider>
```

Comment:
This seemed possibly incorrect, using the provider and passing `darkMode` directly. Saw various patterns here so added the provider, but would love some feedback on this

Reply:
Generally you shouldn't need to do this if whatever `Instance` component reads from context correctly

Reply:
(There is a case for doing something similar if you want to test how the `Instance` responds to the provider _and_ to an override prop. You'd pass different variables in this case then)